packages/nextjs/app/builders/0xb36faaA498D6E40Ee030fF651330aefD1b8D24D2/page.tsx

```diff
@@ -0,0 +1,214 @@
+          </p>
+
+          <div className="space-y-3">
+            <h3 className="text-xl font-semibold text-base-content">Tech Stack</h3>
+            <div className="flex flex-wrap gap-3">
+              {["Solidity", "TypeScript", "Next.js", "Foundry", "AI Agents Framework", "Rust", "Cairo"].map(tech => (
+                <span
+                  key={tech}
+                  className="px-4 py-2 text-base bg-base-200 rounded-lg text-base-content/70 hover:text-primary hover:bg-base-300 transition-all duration-300 hover:scale-105"
+                >
+                  {tech}
+                </span>
+              ))}
+            </div>
+          </div>
+        </div>
+
+        {/* Certifications Grid */}
+        <div className="bg-base-100 rounded-2xl p-8 shadow-lg border border-base-300">
+          <h2 className="text-2xl font-bold text-base-content mb-6 flex items-center gap-2">
+            <HiAcademicCap className="text-primary" />
+            Blockchain Certifications
+          </h2>
+          <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+            {certifications.map(({ title, issuer, icon: Icon }) => (
+              <div
+                key={title}
+                className="bg-base-200 rounded-xl p-5 hover:bg-base-300 transition-all duration-300 hover:scale-[1.02] group"
+              >
+                <div className="flex items-start gap-4">
+                  <Icon className="w-6 h-6 text-primary group-hover:text-secondary transition-colors" />
+                  <div>
+                    <h3 className="font-semibold text-base-content">{title}</h3>
+                    <p className="text-sm text-base-content/70 mt-1">{issuer}</p>
+                  </div>
+                </div>
+              </div>
+            ))}
+          </div>
+        </div>
+
+        {/* Achievements Section */}
+        <div className="bg-base-100 rounded-2xl p-8 shadow-lg border border-base-300">
+          <h2 className="text-2xl font-bold text-base-content mb-6 flex items-center gap-2">
+            <GiTrophy className="text-primary" />
+            Web3 Hackathons
+          </h2>
+          <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+            {achievements.map(({ title, description, icon: Icon }) => (
+              <div
+                key={title}
+                className="bg-base-200 rounded-xl p-5 hover:bg-base-300 transition-all duration-300 hover:scale-[1.02] group"
+              >
+                <div className="flex items-start gap-4">
+                  <Icon className="w-6 h-6 text-primary group-hover:text-secondary transition-colors" />
+                  <div>
+                    <h3 className="font-semibold text-base-content">{title}</h3>
+                    <p className="text-sm text-base-content/70 mt-1">{description}</p>
+                  </div>
+                </div>
+              </div>
+            ))}
+          </div>
+        </div>
```

Comment:
"Blockchain Certifications" and "Web3 Hackathons" have a very similar look and code. Let's try to reuse the code so we can apply changes in a single place.